ui/simul/src/view/results.js

```diff
@@ -1,31 +1,89 @@
 var m = require('mithril');
+var pairings = require('./pairings');
 var status = require('game').status;
+var filterGames = filterAll;
+
+function filterAll(p) {
+  return p;
+}
+
+function filterPlaying(p) {
+  return p.game.status < status.ids.mate;
+}
+
+function filterWins(p) {
+  return p.wins === false;
+}
+
+function filterDraws(p) {
+  return p.game.status >= status.ids.mate && p.wins === null;
+}
+
+function filterLosses(p) {
+  return p.wins === true;
+}
 
 module.exports = function(ctrl) {
-  return m('div.results', [
-    m('div', [
-      m('div.number', ctrl.data.pairings.filter(function(p) {
-        return p.game.status < status.ids.mate;
-      }).length),
-      m('div.text', 'Playing')
+  return [m('div.results', [
+    m('div',[
+      m('div.button', {
+        onclick: function(){
+          filterGames = filterAll; 
+          m.redraw.strategy('all');
+        }},
+        [m('div', [
+          m('div.number', ctrl.data.pairings.length),
+          m('div.text', 'All')
+        ])
+      ]),
+    ]),
+    m('div',[
+      m('div.button', {
+        onclick: function(){
+          filterGames = filterPlaying;
+          m.redraw.strategy('all');
+        }},
+        [m('div', [
+          m('div.number', ctrl.data.pairings.filter(filterPlaying).length),
+          m('div.text', 'Playing')
+        ])
+      ]),
+    ]),
+    m('div',[
+      m('div.button', {
+        onclick: function(){
+          filterGames = filterWins;
+          m.redraw.strategy('all');
+        }},
+        [m('div', [
+          m('div.number', ctrl.data.pairings.filter(filterWins).length),
+          m('div.text', 'Wins')
+        ])
+      ]),
```

Comment:
Fixed, I think :P 
Also removed the "playing" button on simul completion as you suggested, which makes sense, hadn't thought about that.
I'm still having a problem with the redrawing of the boards. My guess is that I should have a way to have the button selected and do "m.redraw.strategy('none')" when you click on an already selected button and "m.redraw.strategy('all')" when you click on an unselected button. That's why I thought tabs could be good, but I looked as an example the tabs on the lichess lobby, and it was too complicated for me for now.